cypress/integration/seededFlows/adminFlows/users/memberToolsSection.spec.js

```diff
@@ -0,0 +1,256 @@
+describe('Tools Section', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.get('@user').then((user) => {
+      cy.loginAndVisit(user, '/admin/users');
+    });
+  });
+
+  describe('Show section', () => {
+    it('shows the boxes', () => {
+      cy.get('@user').then(({ username }) => {
+        cy.visitAndWaitForUserSideEffects('/admin/users');
+        cy.findByRole('link', { name: username }).click();
+
+        cy.findByRole('main').within(() => {
+          cy.findByRole('link', { name: /Emails/ }).should('be.visible');
+          cy.findByRole('link', { name: /Notes/ }).should('be.visible');
+          cy.findByRole('link', { name: /Credits/ }).should('be.visible');
+          cy.findByRole('link', { name: /Organizations/ }).should('be.visible');
+          cy.findByRole('link', { name: /Reports/ }).should('be.visible');
+          cy.findByRole('link', { name: /Reactions/ }).should('be.visible');
+        });
+      });
+    });
+  });
+
+  describe('Emails', () => {
+    it('Verifies the email', () => {
+      cy.get('@user').then(({ username }) => {
+        cy.visitAndWaitForUserSideEffects('/admin/users');
+        cy.findByRole('link', { name: username }).click();
+        cy.findByRole('link', { name: /Emails/ }).click();
+
+        cy.findByRole('button', { name: 'Verify Email Ownership' }).as(
+          'verifyEmailOwnership',
+        );
+        cy.get('@verifyEmailOwnership').within((button) => {
+          button.click();
+        });
+        cy.findByTestId('snackbar').should(
+          'have.text',
+          'Verification email sent!',
+        );
+      });
+    });
+
+    it('Sends an email to the user and checks its presence in the history', () => {
+      cy.get('@user').then(({ username }) => {
+        cy.visitAndWaitForUserSideEffects('/admin/users');
+        cy.findByRole('link', { name: username }).click();
+        cy.findByRole('link', { name: /Emails/ }).click();
+
+        // Send email
+        cy.findByRole('textbox', { name: 'Subject' }).type('Hello!');
+        cy.findByRole('textbox', { name: 'Body' }).type('This is an email');
+        cy.findByRole('button', { name: 'Send Email' }).click();
+
+        // Check message coming from the server
+        cy.findByTestId('snackbar').should('have.text', 'Email sent!');
+
+        // Go back to check its presence in the history
+        cy.visitAndWaitForUserSideEffects('/admin/users');
+        cy.findByRole('link', { name: username }).click();
+        cy.findByRole('link', { name: /Emails/ }).click();
+
+        cy.findAllByText(/Emails history/)
```

Comment:
can we target by a role?

Reply:
I couldn't find a suitable role for the `<details>` :(